src/PreCICE.jl

```diff
@@ -314,48 +355,54 @@ end
 
     hasMesh(meshName::String)::Bool
 
-Check if the mesh with given name is used by a solver. 
+Check if the mesh with given name is used by a solver.
+
+# Arguments
+ - `meshName::String`: Name of the mesh.
 """
 function hasMesh(meshName::String)::Bool
     ans::Integer = ccall((:precicec_hasMesh, "libprecice"), Cint, (Ptr{Int8},), meshName)
     return ans
 end
 
+
 @doc """
 
-    requiresMeshConnectivityFor(meshName::String)::Bool
+    hasData(dataName::String, meshName::String)::Bool
+
+Checks if the data with given name is used by a solver and mesh.
+
+# Arguments
+ - `dataName::String`: Name of the data.
+ - `meshName::String`: Name of the associated mesh.
 
-Checks if the given mesh requires connectivity.
 """
-function requiresMeshConnectivityFor(meshName::String)::Bool
+function hasData(dataName::String, meshName::String)::Bool
     ans::Integer = ccall(
-        (:precicec_requiresMeshConnectivityFor, "libprecice"),
+        (:precicec_hasData, "libprecice"),
         Cint,
-        (Ptr{Int8},),
+        (Ptr{Int8}, Ptr{Int8}),
+        dataName,
         meshName,
     )
     return ans
 end
 
-
 @doc """
 
-    hasData(dataName::String, meshName::String)::Bool
-
-Check if the data with given name is used by a solver and mesh.
-Return true if the mesh is already used.
+    requiresMeshConnectivityFor(meshName::String)::Bool
 
-# Arguments
- - `dataName::String`: Name of the data.
- - `meshName::String`: Name of the associated mesh.
+Checks if the given mesh requires connectivity.
 
+preCICE may require connectivity information from the solver and
+ignores any API calls regarding connectivity if it is not required.
+Use this function to conditionally generate this connectivity.
 """
-function hasData(dataName::String, meshName::String)::Bool
+function requiresMeshConnectivityFor(meshName::String)::Bool
```

Comment:
This change is exactly opposite to the change just above. Is this purely to match the ordering of the functions in the original API?